### ISSUE TITLE
Filters disappear after filtering

### ISSUE THREAD

**Issue by govuklaurence** [datagovuk/find_data_beta#155]

**Comment by govuklaurence** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Gauntlet Bug Fixes](https://trello.com/c/i3BEJXyD/353-gauntlet-bug-fixes)

**Comment by nicholsj** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [When you’ve entered a filter, the value should also be displayed in the filter text field after you click "Apply filters"](https://trello.com/c/JVnsyLeJ/370-when-youve-entered-a-filter-the-value-should-also-be-displayed-in-the-filter-text-field-after-you-click-apply-filters)

**Comment by maxf** [comment]
This issue, along with #111 are due to using accessible-autocomplete, which works on 2 types of elements: 
- <div> as we have now -- but that doesn't work without javascript, or
- <select> -- but that fixes the list to a set of values.
We want neither. I've tried using it on an <input> but it hasn't worked.
So either:
- we hack it to make <input> work
- we use something else (in publish-alpha we had the same problem and so we moved twitter-typeahead instead)
